src/ci/citool/src/utils.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(unused)]
```

Comment:
can you write a comment explaining why this is here?

In general, I'm not sure if it's worth adding this complexity. I can imagine us easily introducing bugs because of this feature.

Aren't there other ways to speed up the compilation? E.g. use caching of dependencies.